src/renderer/containers/widgets/channelSettings/__snapshots__/ChannelSettingsModal.test.ts.snap

```diff
@@ -14,7 +14,7 @@ Object {
     "messages": Array [],
     "newMessages": Array [],
     "typingIndicator": false,
-    "username": "",
+    "username": "unknown",
```

Comment:
I think now that we are not using zcash memos we can remove the notion of an unknown username.

Reply:
It's temporary, we lost mapping for old dms, because we don't have old public keys anymore. I've added this unknown to every old dm thread to just display something better than '@'